.github/ISSUE_TEMPLATE/feature_request_beta.yaml

```diff
@@ -0,0 +1,106 @@
+---
+name: ✨ Feature Request Beta
```

Comment:
Are we going to change this to non-beta before the release?

Reply:
This is using the GH Issue Form method. Demoed with #1436. As long as everyone is OK with the slight representation differences, I will promote this before release. Needed to be merged to try it out here: https://github.com/nautobot/nautobot/issues/new?assignees=&labels=type%3A+feature&template=feature_request_beta.yaml

Reply:
Looks good to me - I say go for it.